test/show.jl

```diff
@@ -0,0 +1,73 @@
+using HypothesisTests, Base.Test
+
+# based on power_divergence.jl tests
+d = [[762,484] [327,239] [468,477]]
+m = PowerDivergenceTest(d)
+
+@test sprint(show, m) == """Pearson's Chi-square Test
+-------------------------
```

Comment:
I think the most common convention is to put `"""` on the second line, with four spaces of indentation (just like for any other code), and then align all others lines to `Pearson`. That way it's clear that the code line continues, and the output appears exactly as it's printed on screen.